### ISSUE TITLE
Add attribution to map

### ISSUE THREAD

**Issue by maxammann** [maplibre/maplibre.github.io#69]
I'm not sure if attribution is needed for the main map. We might want/need to attribute MapTiler and the original data source.

Related: https://github.com/maplibre/demotiles/issues/11

**Comment by lseelenbinder** [comment]
I believe the data is all from Natural Earth, so no attribution is necessary. We can optionally add one, of course. :)

**Comment by maxammann** [comment]
That would be great, I wasn't sure if we need to attribute for example OpenMapTilies (or MapTiler Desktop Pro, because it was used for generation).

**Comment by lseelenbinder** [comment]
I don’t think so, but I’m not certain. Even if so, we could a simple tippecanoe command to rebuild it directly from natural earth (if needed).

**Comment by Etheonor** [comment]
Hello everyone!

I have the same question. I'm working on a small project and love the mbtiles generated on maplibre but I'm not sure if there is some limitation to use it. 

The mbtile was generated with tippecanoe?

**Comment by birkskyum** [comment]
@Etheonor , our community Slack would be the better place to ask such question. When/if asking there, if you could clarify if you refer to technical/legal limitations would be great - I don't think there are any legal limitations. Also note, there are two tippecanoe repos on github - I'd recommend using this one as it's more maintained and driven by the original creators of the project: [felt/tippecanoe](https://github.com/felt/tippecanoe).
